common/fpga.make

```diff
@@ -228,14 +232,18 @@ u-boot-src: $(U_BOOT_SRC)
 .PHONY: u-boot-src
 # -----------------------------------------------------------------------------------
 
-$(DEVTREE_DTB): $(SDK_EXPORT) $(TARGET_DTS)
+$(DEVTREE_DTB): $(SDK_EXPORT) $(TARGET_DTS) | $(DEVTREE_DTC)
```

Comment:
What does the `|` mean?

Reply:
Order-only pre-requisite - checks it exists but does not trigger rebuild of DTB if DTC timestamp gets changed. I think this is appropriate as we are not changing the dtc sources, but just need to binary to exist.

Reply:
In which case it looks fine

Reply:
I would suggest not using the `|` for this case, I agree that changing DTC sources is not planned (in which case, the timestamp check doesn't matter), but if we ever have a reason to do it in future, this will get in our way for testing.